CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h

```diff
@@ -73,11 +73,11 @@ struct _NSDictionaryBridge {
     CFIndex (*countForKey)(CFTypeRef dictionary, CFTypeRef key);
     bool (*containsKey)(CFTypeRef dictionary, CFTypeRef key);
     _Nullable CFTypeRef (*_Nonnull objectForKey)(CFTypeRef dictionary, CFTypeRef key);
-    bool (*_getValueIfPresent)(CFTypeRef dictionary, CFTypeRef key, CFTypeRef _Nonnull * _Nullable value);
+    bool (*_getValueIfPresent)(CFTypeRef dictionary, CFTypeRef key, CFTypeRef _Nullable *_Nonnull value);
```

Comment:
Ah, are you permitted to use this to ask if a value is present? Okay, will change. Good catch!